ui/controls.js

```diff
@@ -831,6 +841,44 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     svg.appendChild(spinnerCircle);
   }
 
+  /**
+   * Add fast-forward button on Controls container
```

Comment:
I don't think "fast forward" is the right term for this.
"Fast forward" usually refers to something that changes the playback rate of the video; that's what the element that our UI controls internally refers to as `fast_forward` does.

This is more often called seeking, skipping, or sometimes scrubbing (I think scrubbing is mostly used for seeking via the bar? Not 100% on that, terminology-wise).